pkg/mount/dag_steps.go

```diff
@@ -715,3 +715,42 @@ func (s *State) UKIUnlock(g *herd.Graph, opts ...herd.OpOption) error {
 		return kcrypt.UnlockAll(true)
 	}))...)
 }
+
+// MountLiveCd tries to mount the livecd if we are booting from one into /run/initramfs/live
+// to mimic the same behavior as the livecd on non-uki boot.
+func (s *State) MountLiveCd(g *herd.Graph, opts ...herd.OpOption) error {
+	return g.Add(cnst.OpUkiMountLivecd, append(opts, herd.WithCallback(func(ctx context.Context) error {
+		// If we are booting from Install Media
+		if !internalUtils.EfiBootFromInstall() {
+			err := os.MkdirAll(s.path("/run/initramfs/live"), 0755)
+			if err != nil {
+				internalUtils.Log.Err(err).Msg("Creating /run/initramfs/live")
+				return nil
+			}
+			// Try to find the CDROM device by label /dev/disk/by-label/UKI_ISO_INSTALL
+			_, err = os.Stat("/dev/disk/by-label/UKI_ISO_INSTALL")
```

Comment:
maybe this string can be moved to a constant

Reply:
yep, will do